corgidrp/l2b_to_l3.py

```diff
@@ -30,6 +32,44 @@ def divide_by_exptime(input_dataset):
 
     return input_dataset.copy()
 
+def determine_color_cor(input_dataset, calspec_filepath, source_filepath):
+    """
+    determine the color correction factor of the observed source
+    at the reference wavelength of the used filter band and put it into the header.
+    We assume that each frame in the dataset was observed with the same color filter.
+    
+    Args:
+        input_dataset (corgidrp.data.Dataset): a dataset of Images (L2b-level)
+        calspec_filepath (str): file name of the known reference flux (usually CALSPEC)
+        source_filepath (str): file name of the flux model of the observed source in 
```

Comment:
Do you envision that we'll have a library of possible star spectra that we download alongside the pipeline, and we manually pick the one we should use in the future? 

As a follow-up, is it useful to allow the user to specify a blackbody temperature to instead use a simple blackbody model for this?

Reply:
I guess we should ask this the calibration group, we could also try to access the corresponding calspec data via automatic download. I doubt that a simple BB model is good enough for the flux calibration on sky?

Reply:
I like the idea of automatic download of calspec files as needed for this. Probably don't need to do it for this PR, but we can chalk it up to future work.